doc/langref.html.in

```diff
@@ -9673,9 +9673,19 @@ fn doTheTest() !void {
       {#header_open|@round#}
       <pre>{#syntax#}@round(value: anytype) @TypeOf(value){#endsyntax#}</pre>
       <p>
-      Rounds the given floating point number to an integer, away from zero. Uses a dedicated hardware instruction
-      when available.
+      Rounds the given floating point number to the nearest integer. If two integers are equally close, round away from zero.
+      Uses a dedicated hardware instruction when available.
       </p>
+      {#code_begin|test|test_round_builtin#}
+const expect = @import("std").testing.expect;
+
+test "@round" {
+    try expect(@round(1.4) == 1);
+    try expect(@round(1.5) == 2);
+    try expect(@round(-1.4) == -1);
+    try expect(@round(-1.5) == -2);
```

Comment:
Might be useful to have at leat one test case that demonstrates that it's not rounding half to even, for clarity.
```suggestion
    try expect(@round(-2.5) == -3);
```
This would round to 2 under half to even.